packages/eslint-plugin-next/src/rules/google-font-display.ts

```diff
@@ -1,8 +1,9 @@
-const NodeAttributes = require('../utils/node-attributes.js')
+import { defineRule } from '../utils/define-rule'
+import NodeAttributes from '../utils/node-attributes'
 
 const url = 'https://nextjs.org/docs/messages/google-font-display'
 
-module.exports = {
+export = defineRule({
```

Comment:
Since the `esModuleInterop` is not enabled in the `.swcrc` and `tsconfig.json`, the typescript's `export =` syntax is used to mimic the `module.exports` behavior.

Reply:
I think this should be reverted/use `module.exports` everywhere. See the reasoning here: https://github.com/vercel/next.js/pull/38647#discussion_r956475755

Reply:
But I am not using `export default` here.

I am using a special syntax `export =`: https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

`export =` is how typescript uses to mimic `module.exports` behavior when not using `esModuleInterop`. swc will transpile `export =` into `module.exports =` as well, [check this out on swc playground](https://play.swc.rs/?version=1.3.3&code=H4sIAAAAAAAAA0vOzysuUSgqzUlVsFWorrXmSq0oyC8qAXJAYtZcAMQs3l8gAAAA&config=H4sIAAAAAAAAA0WOQQrDQAhF7%2BI6i7bQTe%2FQQ8jUhAmZcVAHGkLuXg0J3an%2F%2BXSDWRO8NmgoShKVrtXwCy%2BwtZEmyc1gAFMfjbgo7d4IVh1ZSvBCmOwoerVcyDexGxe0nGAPGmUiJ4D0cbs%2FXbYwK526AUqueVzDkLg0IdV%2FhHVaLjJcF3vmWd%2F86UGY9ODPbjue95NuLFxn9Ud%2B40J6UesAAAA%3D)

Reply:
Here is the output I am seeing locally with `pnpm build` :thinking::

![image](https://user-images.githubusercontent.com/18369201/193260051-93889f58-ed8c-4ff7-8348-821da4870095.png)

Reply:
You are right. I was playing with it locally and saw it wrong in the `dist` folder. Looks good then! :+1: